### PR TITLE
metrics-server v0.3.2 build script for ubi 8.4

### PR DESCRIPTION
**Pull request by Sapana-Khemkar**
metrics-server v0.3.2 build script for ubi 8.4

### CODE REVIEW

**Comment by seth-priya** [comment]
@pratham-m please review.

File: m/metrics-server/metrics-server_ubi_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# ----------------------------------------------------------------------------
+#
+# Package        : metrics-server
+# Version        : 
+# Source repo    :
```
Comment (kandarpamalipeddi) [inline m/metrics-server/metrics-server_ubi_8.4.sh:5]: Please update the header as per latest script guidelines.

File: m/metrics-server/metrics-server_ubi_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
```
Comment (kandarpamalipeddi) [inline m/metrics-server/metrics-server_ubi_8.4.sh:18]: Use  `#!/bin/bash -e`

File: m/metrics-server/metrics-server_ubi_8.4.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+#install go
+rm -rf /bin/go
+wget https://golang.org/dl/$GO_VERSION.linux-ppc64le.tar.gz && \
+tar -C /bin -xzf $GO_VERSION.linux-ppc64le.tar.gz  && \
+rm -f $GO_VERSION.linux-ppc64le.tar.gz && \
+
+#set GO PATH
+export PATH=$PATH:/bin/go/bin
+export GOPATH=/home/tester/go
```
Comment (kandarpamalipeddi) [inline m/metrics-server/metrics-server_ubi_8.4.sh:37]: Use 
`export GOPATH=$(go env GOPATH)`  or create the `/home/tester/go` path before using it.

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

**Review by seth-priya — APPROVED** [review]

**Pull request merged**